stdlib/2and3/logging/__init__.pyi

```diff
@@ -29,6 +29,7 @@ raiseExceptions: bool
 logThreads: bool
 logMultiprocessing: bool
 logProcesses: bool
+_srcfile: Optional[Union[str, bytes]]
```

Comment:
When would this be bytes? It seems to be set from `co_filename`, which as far as I know is always a str.

Reply:
Thanks for pointing that out, and sorry for the slow turn around on my part, I've been otherwise busy. I just double-checked and there's a check in the C-API here for code objects for co_filename being null or a PyUnicodeType (str) so I'll be modifying the PR to match this. Originally, I just did a quick check of the type info for os.normcase, but it can be further narrowed here.